src/gen-xml.ts

```diff
@@ -565,7 +565,16 @@ function slideObjectToXml(slide: PresSlide | SlideLayout): string {
 				let sizing = slideItemObj.options.sizing,
 					rounding = slideItemObj.options.rounding,
 					width = cx,
-					height = cy
+					height = cy,
+					overlayXmlString = slideItemObj.options.overlay
```

Comment:
How about extracting this to a function?

Reply:
We should validate the `overlay` options, and potentially set defaults or we could end up with a corrupted pptx

Reply:
I suggest having something similar to `correctShadowOptions` for overlay options, which checks if the passed overlay options are valid and sets defaults + console warning if they're not.